### DISCUSSION TITLE
FDS Liquid "Fuels"

### DISCUSSION THREAD

**Discussion by TasKall67** [firemodels/fds#13117] [General]
Hi there, i would like to simulate a case in which I'll have a burning area (lets say a pool fire of 2m2 that will provide high temperature >400C) and from a distance of 1-2 meter i will blow a liquid. I want to see if the liquid will take fire and how the fire will spread (e.g. spark flames, fireball). But, beside the known flammable fuels such as Benzene, Butane etc. i want to test it with other substances such as oils, silicon oils,  chloroform or event salty water (that's a crazy case but i want to test the method) that are not flammable (or they have low flammability) and they are not hydrocarbons. For example silicon oil chemical formula is like C6H18OSi2. How can I design such a case?

Here is an example of what i have designed so far but it can work only for Hydrocarbons:

&HEAD CHID='Case02_01'/
&TIME T_END=30.0/
&DUMP DT_RESTART=10.0, DT_SL3D=0.25/

&MESH ID='MESH-01', IJK=40,80,120, XB=-40.0,-20.0,-90.0,-50.0,0.0,60.0, MPI_PROCESS=0/
&MESH ID='MESH-02', IJK=40,80,120, XB=-40.0,-20.0,-50.0,-10.0,0.0,60.0, MPI_PROCESS=1/
&MESH ID='MESH-03', IJK=40,80,120, XB=-40.0,-20.0,-10.0,30.0,0.0,60.0, MPI_PROCESS=2/
&MESH ID='MESH-04', IJK=40,80,120, XB=-40.0,-20.0,30.0,70.0,0.0,60.0, MPI_PROCESS=3/
&MESH ID='MESH-05', IJK=40,80,120, XB=-20.0,0.0,-90.0,-50.0,0.0,60.0, MPI_PROCESS=4/
&MESH ID='MESH-06', IJK=40,80,120, XB=-20.0,0.0,-50.0,-10.0,0.0,60.0, MPI_PROCESS=5/
&MESH ID='MESH-07', IJK=40,80,120, XB=-20.0,0.0,-10.0,30.0,0.0,60.0, MPI_PROCESS=6/
&MESH ID='MESH-08', IJK=40,80,120, XB=-20.0,0.0,30.0,70.0,0.0,60.0, MPI_PROCESS=7/
&MESH ID='MESH-09', IJK=40,80,120, XB=0.0,20.0,-90.0,-50.0,0.0,60.0, MPI_PROCESS=8/
&MESH ID='MESH-10', IJK=40,80,120, XB=0.0,20.0,-50.0,-10.0,0.0,60.0, MPI_PROCESS=9/
&MESH ID='MESH-11', IJK=40,80,120, XB=0.0,20.0,-10.0,30.0,0.0,60.0, MPI_PROCESS=10/
&MESH ID='MESH-12', IJK=40,80,120, XB=0.0,20.0,30.0,70.0,0.0,60.0, MPI_PROCESS=11/
&MESH ID='MESH-13', IJK=40,80,120, XB=20.0,40.0,-90.0,-50.0,0.0,60.0, MPI_PROCESS=12/
&MESH ID='MESH-14', IJK=40,80,120, XB=20.0,40.0,-50.0,-10.0,0.0,60.0, MPI_PROCESS=13/
&MESH ID='MESH-15', IJK=40,80,120, XB=20.0,40.0,-10.0,30.0,0.0,60.0, MPI_PROCESS=14/
&MESH ID='MESH-16', IJK=40,80,120, XB=20.0,40.0,30.0,70.0,0.0,60.0, MPI_PROCESS=15/
&MESH ID='MESH-17', IJK=40,80,120, XB=40.0,60.0,-90.0,-50.0,0.0,60.0, MPI_PROCESS=16/
&MESH ID='MESH-18', IJK=40,80,120, XB=40.0,60.0,-50.0,-10.0,0.0,60.0, MPI_PROCESS=17/
&MESH ID='MESH-19', IJK=40,80,120, XB=40.0,60.0,-10.0,30.0,0.0,60.0, MPI_PROCESS=18/
&MESH ID='MESH-20', IJK=40,80,120, XB=40.0,60.0,30.0,70.0,0.0,60.0, MPI_PROCESS=19/

&SPEC ID='LiquidFuel', FYI='Liquid Benzene', FORMULA='C6H6', DENSITY_LIQUID=879.0, SPECIFIC_HEAT_LIQUID=1.5, VAPORIZATION_TEMPERATURE=80.0, MELTING_TEMPERATURE=5.0, HEAT_OF_VAPORIZATION=433.0/

&REAC ID='FDS6 BENZENE_TK',
      FUEL='LiquidFuel',
      CRITICAL_FLAME_TEMPERATURE=1900.0,
      AUTO_IGNITION_TEMPERATURE=400.0,
      CO_YIELD=0.067,
      SOOT_YIELD=0.181,
      HEAT_OF_COMBUSTION=4.01E+4/

&DEVC ID='TIMER->OUT', QUANTITY='TIME', XYZ=-40.0,-90.0,0.0, SETPOINT=1.0/

&SURF ID='AirBlow',
      RGB=26,114,176,
      VEL=-10.0,
      TAU_V=-1.0/
&SURF ID='BlowLiquid',
      RGB=204,204,0,
      MASS_FLUX=600.0,
      SPEC_ID='LiquidFuel',
      RAMP_MF='BlowLiquid_RAMP_MF'/
&RAMP ID='BlowLiquid_RAMP_MF', T=10.0, F=0.0/
&RAMP ID='BlowLiquid_RAMP_MF', T=10.5, F=1.0/
&RAMP ID='BlowLiquid_RAMP_MF', T=11.5, F=1.0/
&RAMP ID='BlowLiquid_RAMP_MF', T=12.0, F=0.0/
&SURF ID='Flame',
      COLOR='RED',
      HEAT_TRANSFER_COEFFICIENT=1.0E+4,
      NET_HEAT_FLUX=1.0E+4,
      RAMP_Q='Flame_RAMP_Q'/
&RAMP ID='Flame_RAMP_Q', T=0.0, F=0.0/
&RAMP ID='Flame_RAMP_Q', T=7.0, F=7.5/
&RAMP ID='Flame_RAMP_Q', T=9.5, F=1.0/
&RAMP ID='Flame_RAMP_Q', T=14.0, F=1.0/
&RAMP ID='Flame_RAMP_Q', T=14.5, F=0.0/

&OBST ID='Barrel', XB=10.5,12.5,48.0,49.5,0.5,2.0/ 
&OBST ID='Obstruction', XB=11.0,12.0,44.5,45.5,5.551115E-17,0.4, SURF_ID='INERT'/ 

&VENT ID='BlowerVent', SURF_ID='BlowLiquid', XB=10.5,12.5,48.0,48.0,0.5,2.0/ 
&VENT ID='Vent01', SURF_ID='OPEN', XB=-40.0,60.0,-90.0,70.0,60.0,60.0/ 
&VENT ID='Vent02', SURF_ID='OPEN', XB=60.0,60.0,-90.0,70.0,0.0,60.0/ 
&VENT ID='Vent03', SURF_ID='OPEN', XB=-40.0,60.0,-90.0,-90.0,0.0,60.0/ 
&VENT ID='Vent04', SURF_ID='OPEN', XB=-40.0,-40.0,-90.0,70.0,0.0,60.0/ 
&VENT ID='Vent05', SURF_ID='AirBlow', XB=-40.0,60.0,70.0,70.0,0.0,60.0/ 
&VENT ID='Sparks', SURF_ID='Flame', XB=11.0,12.0,44.5,45.5,0.4,0.4, DEVC_ID='TIMER->OUT'/ 

&SLCF QUANTITY='TEMPERATURE', ID='Temp', PBY=45.0/
&SLCF QUANTITY='VELOCITY', VECTOR=.TRUE., ID='Air', PBY=15.0/
&SLCF QUANTITY='VELOCITY', VECTOR=.TRUE., ID='AirX', PBX=11.5/
&SLCF QUANTITY='VOLUME FRACTION', SPEC_ID='LiquidFuel', ID='Benzene', PBY=15.0/
&SLCF QUANTITY='VOLUME FRACTION', SPEC_ID='LiquidFuel', ID='Benzene', PBX=11.5/
&SLCF QUANTITY='TEMPERATURE', ID='Slice', PBX=11.5/

&DEVC ID='[Species: LiquidFuel] Mass Flux_MEAN', QUANTITY='MASS FLUX', SPEC_ID='LiquidFuel', SPATIAL_STATISTIC='MEAN', XB=-40.0,60.0,-90.0,70.0,0.0,60.0/
&DEVC ID='[Species: LiquidFuel] Mass Flux_MAX', QUANTITY='MASS FLUX', SPEC_ID='LiquidFuel', SPATIAL_STATISTIC='MAX', XB=-40.0,60.0,-90.0,70.0,0.0,60.0/
&DEVC ID='[Species: LiquidFuel] Mass Fraction_VOLUME INTEGRAL', QUANTITY='MASS FRACTION', SPEC_ID='LiquidFuel', SPATIAL_STATISTIC='VOLUME INTEGRAL', XB=-40.0,60.0,-90.0,70.0,0.0,60.0/
&DEVC ID='[Species: LiquidFuel] Mass Fraction_MASS INTEGRAL', QUANTITY='MASS FRACTION', SPEC_ID='LiquidFuel', SPATIAL_STATISTIC='MASS INTEGRAL', XB=-40.0,60.0,-90.0,70.0,0.0,60.0/

&TAIL /

Thank you in advance

**Comment by drjfloyd** [comment]
13.2 Complex Stoichiometry

**Reply by TasKall67** [reply]
Can you please give me an example for a specific fuel for example pine oil with chemical formula C11H18O? (This substance might be solved with simple chemistry but just for an example. Or if it is easy lets try an oil type).

Thank you in advance

**Reply by drjfloyd** [reply]
There is an example in 13.2. Follow that substituting your specific fuel molecule.

**Reply by TasKall67** [reply]
Great i will take a look. But, from your experience i would like to know if it is possible in fds to simulate such a case and let fds decide how to burn the substance and not let it assumed that the fuel burns immediately when enters tha domain. Correct?

**Reply by drjfloyd** [reply]
There are examples of fuel sprays, where liquid droplets must first evaporate before burning, in both the Verification Guide and the Validation Guide.

**Reply by TasKall67** [reply]
This is exactly what i need -> liquid fuel that needs to evaporate first before burning. But as i see in the spray fuel example it uses simple chemistry model to run the reaction.
I am trying the "example" in section 13.2 (Non-Simple Chemistry Compatible Fuels) for PVC, but when i "blow" PVC in the domain it starts burning immediately. I do not want to burn immediately but let FDS to choose if there are the correct conditions (sparks, heat etc) to start the fuel burning and fire spread. (please check the following code)


&HEAD CHID='Ignition02_PVC'/
&TIME T_END=30.0/
&DUMP DT_RESTART=300.0, DT_SL3D=0.25/

&MESH ID='Mesh01', IJK=25,46,25, XB=0.0,5.0,0.0,9.2,0.0,5.0/

&SPEC ID='NITROGEN', LUMPED_COMPONENT_ONLY=.TRUE./
&SPEC ID='OXYGEN', LUMPED_COMPONENT_ONLY=.TRUE./
&SPEC ID='SOOT', LUMPED_COMPONENT_ONLY=.TRUE./
&SPEC ID='WATER VAPOR', LUMPED_COMPONENT_ONLY=.TRUE./
&SPEC ID='CARBON MONOXIDE', LUMPED_COMPONENT_ONLY=.TRUE./
&SPEC ID='CARBON DIOXIDE', LUMPED_COMPONENT_ONLY=.TRUE./
&SPEC ID='PVC', FORMULA='C2H3Cl', DENSITY_LIQUID=1.4, SPECIFIC_HEAT_LIQUID=1.0, VAPORIZATION_TEMPERATURE=300.0, MELTING_TEMPERATURE=200.0, HEAT_OF_VAPORIZATION=3000.0, ENTHALPY_OF_FORMATION=297.492/
&SPEC ID='HYDROGEN CHLORIDE', FORMULA='HCl', LUMPED_COMPONENT_ONLY=.TRUE./
&SPEC ID='MyAir', BACKGROUND=.TRUE., 
  SPEC_ID(1)='NITROGEN',
  SPEC_ID(2)='OXYGEN', 
  VOLUME_FRACTION(1)=5.76,
  VOLUME_FRACTION(2)=1.53/
&SPEC ID='MyProducts', 
  SPEC_ID(1)='CARBON DIOXIDE',
  SPEC_ID(2)='CARBON MONOXIDE',
  SPEC_ID(3)='HYDROGEN CHLORIDE',
  SPEC_ID(4)='NITROGEN',
  SPEC_ID(5)='SOOT',
  SPEC_ID(6)='WATER VAPOR', 
  VOLUME_FRACTION(1)=0.96,
  VOLUME_FRACTION(2)=0.14,
  VOLUME_FRACTION(3)=1.0,
  VOLUME_FRACTION(4)=5.76,
  VOLUME_FRACTION(5)=0.9,
  VOLUME_FRACTION(6)=1.0/


&REAC ID='Reaction2',
      HEAT_OF_COMBUSTION=1.64E+4,
      FUEL='PVC',
      SPEC_ID_NU='MyAir','PVC','MyProducts',
      NU=-1.0,-1.0,1.0/

&PART ID='Tracer', MASSLESS=.TRUE., MONODISPERSE=.TRUE., RGB=51,204,255, AGE=60.0, DEVC_ID='TIMER->OUT'/
&DEVC ID='TIMER->OUT', QUANTITY='TIME', XYZ=0.0,0.0,0.0, SETPOINT=2.9/

&SURF ID='AirBlow',
      RGB=26,114,176,
      VEL=-10.0,
      TAU_V=-1.0/
&SURF ID='BlowPVC',
      RGB=204,204,0,
      MASS_FLUX=10.0,
      SPEC_ID='PVC',
      RAMP_MF='BlowPVC_RAMP_MF',
      PART_ID='Tracer',
      NPPC=5/
&RAMP ID='BlowPVC_RAMP_MF', T=0.0, F=0.0/
&RAMP ID='BlowPVC_RAMP_MF', T=2.9, F=0.0/
&RAMP ID='BlowPVC_RAMP_MF', T=3.0, F=1.0/
&RAMP ID='BlowPVC_RAMP_MF', T=4.0, F=1.0/
&RAMP ID='BlowPVC_RAMP_MF', T=4.1, F=0.0/

&OBST ID='Obstruction', XB=2.4,2.8,0.8,1.2,0.6,1.0/ 
&OBST ID='Obstruction', XB=2.4,2.8,1.6,1.8,0.0,0.4, SURF_ID='INERT'/ 

&VENT ID='Mesh Vent: Mesh01 [XMAX]', SURF_ID='OPEN', XB=5.0,5.0,0.0,9.2,0.0,5.0/ 
&VENT ID='Mesh Vent: Mesh01 [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,0.0,9.2,0.0,5.0/ 
&VENT ID='Mesh Vent: Mesh01 [YMAX]', SURF_ID='OPEN', XB=0.0,5.0,9.2,9.2,0.0,5.0/ 
&VENT ID='Mesh Vent: Mesh01 [YMIN]', SURF_ID='AirBlow', XB=0.0,5.0,0.0,0.0,0.0,5.0/ 
&VENT ID='Mesh Vent: Mesh01 [ZMAX]', SURF_ID='OPEN', XB=0.0,5.0,0.0,9.2,5.0,5.0/ 
&VENT ID='Vent', SURF_ID='BlowPVC', XB=2.4,2.8,1.2,1.2,0.6,1.0/ 

&SLCF QUANTITY='TEMPERATURE', ID='Slice', PBX=2.6/


&TAIL /

**Reply by drjfloyd** [reply]
Please read the Technical Reference Guide chapter on the combustion model so you understand how FDS treats combstion, ignition, and extinction.

**Comment by mcgratta** [comment]
Look at the sample case `Fires/spray_burner.fds`. That demonstrates a case where nozzles spray heptane droplets into a pan. The fuel evaporates from the droplets both in the air and in the pan. 

All this being said, I would be cautious about making any conclusions about the different fuels unless you have detailed info about their liquid and gaseous properties, and even then, the combustion is simplified and you may not be able to distinguish one fuel from another except via its basic properties like boiling temperature and heat of combustion..

**Reply by TasKall67** [reply]
For now, i do not want accurate results, just to accomplish the methodology and try a few different species to validate it. I i said in my initial post, i want to check real fuels (benzene, butane etc), non flammable liquids (salty water e.g.) and then other oils
